exporters/otlp/testing-internal/src/main/java/io/opentelemetry/exporter/otlp/testing/internal/ManagedChannelTelemetryExporterBuilder.java

```diff
@@ -151,4 +157,62 @@ public CompletableResultCode shutdown() {
       }
     };
   }
+
+  /**
+   * Configure the channel builder to trust the certificates. The {@code byte[]} should contain an
+   * X.509 certificate collection in PEM format.
+   *
+   * @throws SSLException if error occur processing the certificates
+   */
+  private static void setSslContext(
+      ManagedChannelBuilder<?> managedChannelBuilder, TlsConfigHelper tlsConfigHelper)
+      throws SSLException {
+    requireNonNull(managedChannelBuilder, "managedChannelBuilder");
```

Comment:
The caller already checks this, and there's only one way into this private method, so I'd remove the extra `requireNonNull` here.